app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -85,9 +76,20 @@ def poll_ansible_tower_job_status(interval)
     queue_signal(:post_ansible_run, err.message, 'error')
   end
 
-  def post_ansible_run(*args)
-    # delete inventory, job_template, job?
-    queue_signal(:finish, *args)
+  def post_ansible_run(message, status)
+    inventory_not_deleted = !delete_inventory
+    jt_not_deleted = !delete_job_template
+
+    message = "#{message}; Cleanup encountered error" if inventory_not_deleted || jt_not_deleted
+    signal(:finish, message, status)
+  end
+
+  def playbook
+    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook.find(options[:playbook_id])
```

Comment:
Is `options[:playbook_id]` the Ansible Tower Playbook ID (ems_ref) or the ConfigurationScriptPayload ID?

Reply:
ConfigurationScriptPayload ID